core/trino-main/src/test/java/io/trino/sql/planner/iterative/rule/TestPushProjectionThroughJoin.java

```diff
@@ -114,6 +118,8 @@ public void testPushesProjectionThroughJoin()
     @Test
     public void testDoesNotPushStraddlingProjection()
     {
+        Session session = testSessionBuilder().build();
```

Comment:
Please format this method and the following method consistently wrt added blank lines. 
Or actually, you could just inline `session` in both.
The same comment applies to `TestJoinNodeFlattener`.